Simplify processing workflow request

jraddaoui opened the pull request:
- Remove configuration values already accessible from the workflow.
- Remove SIPID, trust only SIPUUID and remove unused SIP update.

Refs #1210.

djjuhasz commented on internal/ingest/workflows.go:
```diff
@@ -53,29 +49,20 @@ type ProcessingWorkflowRequest struct {
+func InitProcessingWorkflow(
+	ctx context.Context,
+	tc temporalsdk_client.Client,
+	tq string,
```
I think it's worth calling this variable "taskQueue" to make its purpose clear — it's part of the function signature and the "string" type doesn't provide any hints about the argument's purpose.

djjuhasz reviewed (approved):
LGTM @jraddaoui. I just had one quibble about an argument name. :+1:

jraddaoui committed:
Simplify processing workflow request

- Remove configuration values already accessible from the workflow.
- Remove SIPID, trust only SIPUUID and remove unused SIP update.
